tests/unit/test_arg_helpers.py

```diff
@@ -0,0 +1,256 @@
+        arg_helpers.help_with_ops(mock_ops, mocked_config)
+        captured = capsys.readouterr()
+        assert "testkey1" in captured.out
+
+    def test_help_with_ops_with_plugins(self, capsys, mocker, mocked_config):
+        mock_ops = {"testkey1": "testvalue1"}
+        mocker.patch(
+            "linodecli.arg_helpers.plugins.available",
+            return_value=["testing.plugin"],
+        )
+        arg_helpers.help_with_ops(mock_ops, mocked_config)
+        captured = capsys.readouterr()
+        assert "testing.plugin" in captured.out
+
+    # arg_helpers.action_help(cli, command, action)
+    def test_action_help_value_error(self, capsys, mock_cli):
+        arg_helpers.action_help(mock_cli, None, None)
+        captured = capsys.readouterr()
+        assert not captured.out
+
+    def test_action_help_post_method(self, capsys, mocker, mock_cli):
+        mocked_ops = mocker.MagicMock()
+        mocked_ops.summary = "test summary"
+        mocked_ops.docs_url = "https://website.com/endpoint"
+        mocked_ops.method = "post"
+
+        mocked_args = mocker.MagicMock()
+        mocked_args.required = True
+        mocked_args.path = "path"
+        mocked_args.description = "test description"
+
+        mocked_ops.args = [mocked_args]
+
+        mock_cli.find_operation = mocker.Mock(return_value=mocked_ops)
+
+        arg_helpers.action_help(mock_cli, "command", "action")
+        captured = capsys.readouterr()
+        assert "test summary" in captured.out
+        assert "API Documentation" in captured.out
+        assert "https://website.com/endpoint" in captured.out
+        assert "Arguments" in captured.out
+        assert "test description" in captured.out
+        assert "(required)" in captured.out
+        assert "filter results" not in captured.out
+
+    def test_action_help_get_method(self, capsys, mocker, mock_cli):
+        mocked_ops = mocker.MagicMock()
+        mocked_ops.summary = "test summary"
+        mocked_ops.docs_url = "https://website.com/endpoint"
+        mocked_ops.method = "get"
+        mocked_ops.action = "list"
+        mocked_ops.args = None
+
+        mock_attr = mocker.MagicMock()
+        mock_attr.filterable = True
+        mock_attr.name = "filtername"
+        mocked_ops.response_model.attrs = [mock_attr]
+
+        mock_cli.find_operation = mocker.Mock(return_value=mocked_ops)
+
+        arg_helpers.action_help(mock_cli, "command", "action")
+        captured = capsys.readouterr()
+        assert "test summary" in captured.out
+        assert "API Documentation" in captured.out
+        assert "https://website.com/endpoint" in captured.out
+        assert "Arguments" not in captured.out
+        assert "filter results" in captured.out
+        assert "filtername" in captured.out
+
+    def test_bake_command_bad_website(self, capsys, mocker, mock_cli):
+        with pytest.raises(SystemExit) as ex:
+            arg_helpers.bake_command(mock_cli, "https://website.com")
```

Comment:
Are we using mocking here?

Reply:
Ah good catch, i was using it when i wrote it originally but realized i can just not mock it since the bad use of the function gives me the correct systemexit response anyway.